packages/x-tailwindcss/src/x-tailwind-plugin/components/typography/title2/sizes.ts

```diff
@@ -0,0 +1,24 @@
+import { TailwindHelpers } from '../../../../types';
+
+/**
+ * Returns the `sizes` variants for component `title2`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function titleSizes({ theme }: TailwindHelpers) {
+  return {
+    sm: {
+      fontSize: theme('fontSize.md'),
```

Comment:
I still see differences between the browser and the designs. The browser says 18px but on figma I see  18px. 

![image](https://user-images.githubusercontent.com/7504736/207006699-9899fa32-ea2e-4657-8738-bc6e775e1960.png)

Reply:
I was taking a look at the values and I think :belt: got the correct ones based on the tokens definition.
![image](https://user-images.githubusercontent.com/68222542/207008258-493df48e-fac9-47f5-805c-c44a4eb67450.png)

Reply:
fixed in the designs